api/professor.py

```diff
@@ -98,3 +99,29 @@ def delete_professor(professor_id):
         return f'Unable to delete prof with id {professor_id}', 500
 
     return f'Deleted prof with id {professor_id}', 200
+
+@PROFESSOR_BP.route('/remind/<professor_id>/', methods=['POST'])
```

Comment:
I'm so sorry this is super nitpicky, could we move the prof_id before remind so it looks like 
`/<professor_id>/remind/`
Reason being all other api endpoints extend from `/professor/<professor_id> `

Reply:
@itsazoo this should be done now